src/en/authors-charm-metadata.md

```diff
@@ -104,5 +104,45 @@ and whether it's designed for deployment as a
   - if the charm is subordinate, it must contain at least one `requires`
     relation with container scope.
 
+`resources` allows you to add a blob that your charm can utilize.
+
+```yaml
+resources:
+  example:
+    type: file # "file" is the only type supported currently
+    filename: example.tar.gz
+    description: example resource
+```
+
+`payloads` allows you to register payloads such as LXC, KVM, and docker with
+Juju. This lets the operator better understand the purpose and function of these
+payloads on a given machine.
+
+```yaml
+payloads:
+    monitoring:
+        type: docker
+    kvm-guest:
+        type: kvm
+```
+
+`extra-bindings` represents an extra bindable endpoint that is not a relation.
+
+```yaml
+extra-bindings:
+  endpoint1:
+  endpoint2:
+```
+
+Endpoint names are strings and must not match existing relation names from
+the Provides, Requires, or Peers metadata sections. The values beside each
+endpoint name must be left out (i.e. "foo": <anything> is invalid).
```

Comment:
Brackets in output do not render in the computed markdown document. Either remove them, or set this to &lt; &gt;